health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/utils/utilities.js

```
@@ -97,6 +99,48 @@ const formValidator = (formData, key, state) => {
     return null;
   };
 
+  const areFieldsValid = (fields) => {
+    const hasInvalidField = fields?.some(field => {
+      const isValid = field && field.value; 
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)